cfpb/cfgov-refresh
# Docker Compose based developer workflow

Provides a docker-compose based developer workflow.

Check out the instructions for [Installation](https://github.com/cfpb/cfgov-refresh/blob/10minutes/docs/installation.md#docker-compose-installation) and [Usage](https://github.com/cfpb/cfgov-refresh/blob/10minutes/docs/usage.md#usage-docker)

## Additions

- configuration and documentation for Docker and Docker Compose
- a script for creating an initial Docker Machine VM.
- Scripts that ease interacting with the Docker/Compose environment

## Testing

- Verify that checking out this branch does not break your existing workflow. Everything you do on a normal basis should continue to work. If it does, please report it here.
- Try out the installation instructions.
- Read the usage instructions.

## Todos

- This setup presumes that you still have Node, gulp, etc installed on the mac side. I think with further work, we can eliminate that dependency.
- We need a mechanism for SSL server.

## Checklist
